en/tutorials-and-examples/cms/database.rst

```diff
@@ -0,0 +1,107 @@
+CMS Tutorial - Creating the Database
+####################################
+
+Now that we have CakePHP installed, let's set up the database for our :abbr:`CMS
+(Content Management System)` application. If you haven't already done so, create
+an empty database for use in this tutorial, with a name of your choice, e.g.
+``cake_cms``. You can execute the following SQL to create the necessary
+tables::
+
+    CREATE TABLE users (
+        id INT AUTO_INCREMENT PRIMARY KEY,
+        email VARCHAR(255) NOT NULL,
+        password VARCHAR(255) NOT NULL,
+        created DATETIME,
+        modified DATETIME
+    );
+
+    CREATE TABLE articles (
+        id INT AUTO_INCREMENT PRIMARY KEY,
+        user_id INT NOT NULL,
+        title VARCHAR(255) NOT NULL,
+        slug VARCHAR(255) NOT NULL,
+        body TEXT,
+        published BOOLEAN DEFAULT FALSE,
+        created DATETIME,
+        modified DATETIME,
+        UNIQUE KEY (slug),
+        FOREIGN KEY user_key (user_id) REFERENCES users(id)
+    ) CHARSET=utf8mb4;
```

Comment:
I like the idea of pushing people to use `utf8mb4`but I think that if we do it, we need to specify it in the user table also.
I mainly use MySQL but is this supported in other DB (maybe stupid question) but I remember we had such comment in the bookmarker tutorial that was not compliant to every DB we support and that why we pushed people to use migrations ? I would be fine only providing this command or one DB, but maybe a warning on top of it is necessary then.

Also here for example I think that for mySQL the correct statement is `CHARSET SET utf8mb4`